Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM woorank/docker-node-babel
+EXPOSE 8080-8082
+
+ADD node_modules/ /node_modules
```

Comment:
only for tchannel, and when other our discovered I will add them separately to the docker file.